nvram.c

```diff
@@ -14,7 +14,6 @@
 #include <sys/types.h>
```

Comment:
Would you consider rewriting this library to pass all its logic through to pypanda? If I'm reading this correctly, it seems like you've extended it to also do hypercalls (and thus give pypanda a chance to interpose on gets/sets), but the majority of the logic is still happening inside the guest - would be awesome to move this to all live in pypanda